### PR TITLE
Update HandleBars version. Bump up CMake Tools Release version to #1.2.3

### PR DESCRIPTION
**Pull request by andreeis**
Update HandleBars to #4.5.3, because of security reasons.

Update CMake Tools version and changelog for #1.2.3 bug fixes release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # What's New?
 
+## 1.2.3
+Bug fixes:
+- CTest status bar button text appears malformed. [#911] (https://github.com/microsoft/vscode-cmake-tools/issues/911)
```
Comment (bobbrow) [inline CHANGELOG.md]: there should not be a space between `[]` and `()`.  If you put a space there, the hyperlink will not show up correctly.  Can you please fix this and the other ones below (including for 1.2.2.)?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+- Cleanup fix for message "Platform undefined / toolset {}". [#913] (https://github.com/microsoft/vscode-cmake-tools/issues/913)
+- Fix incorrect file associations when language is unset. [#926] (https://github.com/microsoft/vscode-cmake-tools/issues/911)
+
+Improvements:
+- Add telemetry for crashes.
```
Comment (bobbrow) [inline CHANGELOG.md]: I don't think this needs to be listed.

File: CHANGELOG.md (hunk 1 above)
Comment (andreeis) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 2 above)
Comment (andreeis) [reply CHANGELOG.md]: Sure.

**Pull request merged**